docs/network-policy-recommendation.md

```diff
@@ -0,0 +1,129 @@
+# Network Policy Recommendation
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Deployment](#deployment)
+- [Command-line Usage](#command-line-usage)
+  - [Start a policy recommendation job](#start-a-policy-recommendation-job)
+  - [Check the status of a policy recommendation job](#check-the-status-of-a-policy-recommendation-job)
+  - [Get the result of a policy recommendation job](#get-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Network policy recommendation is a feature that utilizes the flow records collected by
```

Comment:
I feel better to talk about what policy recommendation does, before how.

So, we can say something like "Theia NetworkPolicy Recommendation recommends the NetworkPolicy configuration to secure Kubernetes network and applications". And then "It analyzes the network flows collected by Grafana Flow Collector to generate Kubernetes NetworkPolicies and Antrea NetworkPolicies".

Assume @salv-orlando could have some good ideas.

Reply:
I added some comments on  doc PR #36.
@dreamtalen  should this commit be amended so that documentation is entirely contained in PR #36?